src/app/globals.css

```
@@ -100,120 +100,100 @@ html {
+
+/* Word-wrap toggle */
+.cb-root[data-wrap="true"] .cb-scroll {
+  overflow-x: hidden;
 }
 
-.token.entity {
-  cursor: help;
+.cb-root[data-wrap="true"] .shiki code,
+.cb-root[data-wrap="true"] .shiki .line {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)